accessibility/README.md

```diff
@@ -0,0 +1,495 @@
+
+## Table on contents
+
+* [Native Elements](#native-elements)
+* [Keyboard Support](#keyboard-support)
+* [Accessible Text](#accessible-text)
+* [Alt Text](#alt-text)
+* [Element IDs](#element-ids)
+* [Additional Resources](#additional-resources)
+
+[← Back to homepage](../README.md)
+
+## Native Elements
+
+* [Use `<button>` elements for clickable items](#use-button-element-for-clickable-items)
+* [Don't use `<a>` elements where a `<button>` would be better](#dont-use-a-elements-where-a-button-would-be-better)
+* [Use input events rather than wrapping click events](#use-input-events-rather-than-wrapping-click-events)
+
+### Use `<button>` element for clickable items
+
+#### Don't
+
+```vue
+<div
+  class="dropdown-menu__input"
+  @click="toggleMenu"
+>
+  <span v-text="label">
+  <icon-chevron-down />
+</div>
+
+<responsive-image
+  @click="handleImageClick"
+/>
+
+<a @click="toggleItem">
+  Click here
+</a>
+```
+
+#### Do
+
+```vue
+<button
+  class="dropdown-menu__input"
+  @click="toggleMenu"
+>
+  <span v-text="label">
+  <icon-chevron-down />
+</button>
+
+
+<button @click="handleImageClick">
+  <responsive-image />
+</button>
+
+```
+  
+* Prefer using native `<button>` element instead of adding click event handlers to non-interactive elements like `<div>`
+
+#### Why
+
+* The native `<button>` element comes with accessible defaults out of the box such as:
+  * Keyboard interaction (buttons can be triggered with `Space` or `Enter` keys)
+  * Keyboard focusable
+  * Buttons are declared correctly to screen-readers
+  * A button's interactivity can be natively disabled with the `disabled` attribute
+* Re-creating all of these out of the box defaults on a `<div>` or other element would require additional attributes and event handlers which results in more code to write and more code shipped to the user - Using native elements has performance benefits too.
+* E.g. here's a comparison of using a `<div>` element vs a `<button>` element:
+```vue
+// Bad
+<div
+  role="button"
+  tabindex="0"
+  @click="handleClick"
+  @keydown="handleKeyDown"
+>
+  Click me
+</div>
+
+// Good
+<button @click=:handleClick>
+  Click me
+</button>
+```
+  * Note: This `<div>` in the example does not include additional features that the `<button>` comes with, such as indicating a `disabled` state, the code for `handleKeyDown` so that only responds to `Enter` and `Tab` 
+
+
+### Don't use `<a>` elements where a `<button>` would be better
+
+#### Don't
+
+```vue
+<a @click="toggleItem">
+  Click here
+</a>
+
+<a href="#" @click.prevent="doSomething">
+  Do Something
+</a>
+```
+
+#### Do
+
+```vue
+<button @click="toggleItem">
+  Click here
+</button>
+
+<button @click.prevent="doSomething">
+  Do something
+</button>
+```
+
+* If an element is not linking somewhere, it's better to use a `<button>` element instead
+  * `<a>` tags should not be used without a `href` attribute or with blank or javascript `href` attribute
```

Comment:
```suggestion
* `<a>` tags should not be used without a `href` attribute or with blank or javascript `href` attribute
```
I'd say this bullet is a 'sub' bullet of the preceding one (it's not really related) so can be outdented.